### DISCUSSION TITLE
What is the difference between "image_ROC.png" and "pixel_ROC.png"?

### DISCUSSION THREAD

**Discussion by William9Baker** [open-edge-platform/anomalib#1205] [Q&A] [answered]

**Comment by blaz-r** [comment, marked as answer]
Hello.
Image_ROC is score per image. This means that entire image is treated as one object, so score is assigned to it, which is then thresholded and compared to it's label (which is usually either 1 - nominal or 0 - anomalous).

Pixel_ROC is score per every pixel of the image. This means that we do similar comparison as in the case above, but each pixel is treated as one object. This means that it is thresholded and then compared to label, which is in this case a pixel in mask (again usually 0 where underlying pixel is good and 1 if it's anomalous). This is then applied over all pixels over all images.

Hope this helps 😃
